dotCMS/src/main/java/com/dotmarketing/business/DeterministicIdentifierAPI.java

```diff
@@ -22,6 +24,10 @@
      */
     String generateDeterministicIdBestEffort(Versionable asset, Treeable parent);
 
+    @CloseDBIfOpened
```

Comment:
Remove this, has not any sense to annotate the interface

Reply:
Done